src/utils/utilsTests/itemFilterUtils.test.tsx

```diff
@@ -71,53 +73,75 @@ describe("Item Filter Utils", () => {
         format: ["Text"],
         location: ["loc:abc", "Offsite"],
         status: ["status:a", "status:na"],
+        year: [],
       }
       const recapLocations = "loc:rc2ma,loc:rc3ma,loc:rc4ma"
-      expect(buildItemFilterQueryString(query, recapLocations)).toBe(
-        "?item_location=loc:abc,loc:rc2ma,loc:rc3ma,loc:rc4ma&item_format=Text&item_status=status:a,status:na"
-      )
+      expect(buildItemFilterQuery(query, recapLocations)).toStrictEqual({
+        item_location: "loc:abc,loc:rc2ma,loc:rc3ma,loc:rc4ma",
+        item_format: "Text",
+        item_status: "status:a,status:na",
+      })
     })
     it("can handle only one param", () => {
       const query = {
         format: [],
         location: ["loc:abc", "Offsite"],
         status: [],
+        year: [],
       }
       const recapLocations = "loc:rc2ma,loc:rc3ma,loc:rc4ma"
-      expect(buildItemFilterQueryString(query, recapLocations)).toBe(
-        "?item_location=loc:abc,loc:rc2ma,loc:rc3ma,loc:rc4ma"
-      )
+      expect(buildItemFilterQuery(query, recapLocations)).toStrictEqual({
+        item_location: "loc:abc,loc:rc2ma,loc:rc3ma,loc:rc4ma",
+      })
     })
   })
 
-  describe("buildAppliedFiltersString", () => {
+  describe("buildAppliedFiltersTagSetData", () => {
     const query = parseItemFilterQueryParams({
       item_location: "loc:rc2ma,loc:rcma2",
       item_status: "status:a",
       item_format: "Text",
+      item_date: "2005",
     })
     const emptyQuery = parseItemFilterQueryParams({})
-    const aggs = normalAggs.map((agg) => new ItemFilterData(agg))
+    const aggregations = normalAggs.map(
+      (aggregation) => new ItemFilterData(aggregation)
+    )
     it("no filters", () => {
-      expect(buildAppliedFiltersString(emptyQuery, aggs)).toBeUndefined()
+      expect(
+        buildAppliedFiltersTagSetData(emptyQuery, aggregations)
+      ).toStrictEqual([])
     })
     it("with all filters", () => {
-      expect(buildAppliedFiltersString(query, aggs)).toBe(
-        "Filtered by location: 'Offsite', format: 'Text', status: 'Available'"
-      )
-    })
-    it("all filters filters", () => {
-      expect(buildAppliedFiltersString(query, aggs)).toBe(
-        "Filtered by location: 'Offsite', format: 'Text', status: 'Available'"
-      )
+      expect(buildAppliedFiltersTagSetData(query, aggregations)).toStrictEqual([
```

Comment:
This and the next test need to be updated with ids that match the labels

Reply:
That's what i thought too when getting ready to open the PR but after I made the change it broke the item clearing functionality.

The way the TagSet component currently works, clicking on a tag passes the id as an attribute, so that needs to match the filter value (instead of the label) so we know which filter needs to be removed in removeValueFromFilters.